client/app/routes/contacts.coffee

```diff
@@ -45,4 +45,9 @@ module.exports = class ContactsRouter extends Backbone.SubRoute
 
 
     duplicates: ->
-        @_ensureContacts -> app.model.set 'dialog', 'duplicates'
+        @_ensureContacts ->
+            $($('nav a').get(1)).attr 'aria-busy', true
```

Comment:
I think we can just toggle the status on the button when clicked, than use a channel event (see below) to toggle it again after. This way we won't deal with view actions in the router.